src/components/common/productCard/productCard.tsx

```diff
@@ -0,0 +1,16 @@
+import {CardProps} from "../../../types";
+import {useParams} from "react-router-dom";
+import api from "../../../api/api.json"
+import {Card} from "../card";
+import css from "./productcCard.module.css"
+
+export const ProductCard = () => {
+
+    const {product} = useParams()
+    const sort = api.find((item: CardProps)=> item.label === product)
```

Comment:
это товар, а не сортировка
```suggestion
    const good = api.find((item: CardProps)=> item.label === product)
```